package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-legacy-blueprints",
-  "version": "0.1.5",
+  "version": "0.2.0",
```

Comment:
theoretically we could do a major bump

Reply:
why? we're sub-1.0 so minor bump is enough for breaking changes